spec/policies/kudo_policy_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+describe KudoPolicy do
+  subject(:kudo_policy) { described_class }
+
+  let(:employee1) { create(:employee) }
+  let(:employee2) { create(:employee) }
+  let!(:kudo) { create(:kudo, giver: employee1) }
+
+  permissions :edit?, :update?, :destroy? do
+    it 'grants access if employee is a giver' do
+      expect(kudo_policy).to permit(employee1, kudo)
+    end
+
+    it 'denies access if employee is not a giver' do
+      expect(kudo_policy).not_to permit(employee2, kudo)
+    end
+
+    it 'denies access if kudo created more than 5 minutes ago' do
```

Comment:
+ test for checking whe kudo can be deleted/updated

Reply:
added 🙂